ReactQt/tests/test-integration.cpp

```diff
@@ -25,30 +27,36 @@ private slots:
     void cleanupTestCase();
 
     void testTestModuleMarkTestCompleted();
+    void testJSExceptionReceived();
 };
 
 void TestIntegration::initTestCase() {
```

Comment:
`initTestCase()` and `cleanupTestCase()` can be omitted, since they are virtual.

Reply:
@vkjr , that's a good point, but it seems QTest requires these methods to be defined explicitly in each class passed to QTEST_MAIN macro.
I was not able to get base class virtual methods called.
And more over, it seems, others test cases has similar issue, that `cleanupTestCase` is not get called at all (since `initTestCase()` is redefined, but  `cleanupTestCase` not ).

Reply:
My best guess, is that QTest lib just defines it's own version of `initTestCase()` and `cleanupTestCase()`  if they are not defined in class.

Reply:
@MaxRis, interesting info, thanks!